compiler/one-cmds/CMakeLists.txt

```diff
@@ -42,13 +67,13 @@ if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "aarch64")
   #      and install build/../one-prepare-venv file
   list(APPEND ONE_COMMAND_FILES one-prepare-venv.aarch64)
 else(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL "aarch64")
-  if(ONE_UBUNTU_CODENAME_BIONIC)
+  if(ONE_UBUNTU_CODENAME_BIONIC OR ONE_UBUNTU_CODENAME_NOBLE)
```

Comment:
maybe we should add `one-prepare-venv.u2204` not to get confused ?

Reply:
I think two options
- Rename `one-prepare-venv.u1804` to `one-prepare-venv.py38` because it uses python 3.8
- New file `one-prepare-venv.u2404` as you suggested 

But I also think option 2 (`one-prepare-venv.u2404`) looks better. I'll update if you don't have any objection.